contracts/FindMarketAuctionSoft.cdc

```diff
@@ -113,11 +113,11 @@ pub contract FindMarketAuctionSoft {
 			self.auctionEndsAt=endsAt
 		}
 
-		pub fun hasAuctionEnded() : Bool {
+		pub fun hasAuctionEnded() : Bool? {     // nil means this auction is not live
```

Comment:
Would prefer a method alled isAuctionOngoing and this just returning boolean and calling it hasOngoingAuctionEnded